<launch>

  <!-- <rosparam file="$(find dr_one_setup)/param/tf_params.yaml" command="load"/> -->

  <param name="dr_one" textfile="$(find dr_one_setup)/urdf/quadrotor_base.urdf" />

  <!-- <node pkg="tf" name="tf_static_vodom" type="static_transform_publisher" args="0 0 0 0 0 0 /vodom /base_link 2" /> -->
  <node pkg="tf" name="tf_static_laser" type="static_transform_publisher" args="0 0 0.07 0 0 0 /base_link /laser 2" />
  <node pkg="dr_one_setup" name="laser_tf_listener" type="laser_tf_listener"/>
    <!-- <node pkg="dr_one_setup" name="tf_broadcaster" type="tf_broadcaster" args="/tf_broadcaster"/> -->
  <node pkg="rviz" name="rviz" type="rviz" args="-d $(find dr_one_setup)/test_sim.rviz" />

</launch>
